examples/python-flask-starter-with-oauth/README.md

```diff
@@ -1,44 +1,80 @@
-### About the app
+# Miro Flask Python Starter with OAuth
 
-This app shows how you can use the Miro REST API. \
-It implements the full OAuth2 flow and calls the [Create Miro board API endpoint](https://developers.miro.com/reference/create-board).
+This app shows how to implement the OAuth2.0 flow and call the Miro REST API's [Create Board](https://developers.miro.com/reference/create-board) endpoint, using Flask / Python.
 
-This app uses [Flask](https://flask.palletsprojects.com/en/2.1.x/) and [Python](https://www.python.org/).
+# 👨🏻‍💻 App Demo
 
-## How to start
+https://github.com/miroapp/app-examples/assets/10800544/767dba77-c3dd-40b0-b6d7-9b216f0fb0b8
 
-1. Install the project dependenciesAwes: run `pip3 install -r requirements.txt`
-2. Create your `.env` file by copying the template, and use it to store your credentials: \
+# 📒 Table of Contents
+
+- [Included Features](#features)
+- [Tools and Technologies](#tools)
+- [Prerequisites](#prerequisites)
+- [Run the app locally](#run)
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
+
+# ⚙️ Included Features <a name="features"></a>
+
+- [Miro REST API](https://developers.miro.com/reference/api-reference)
+- [Miro OAuth2.0](https://developers.miro.com/docs/getting-started-with-oauth)
+
+# 🛠️ Tools and Technologies <a name="tools"></a>
+
+- [Flask](https://flask.palletsprojects.com/en/2.1.x/)
+- [Python](https://www.python.org/)
+
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- You have the latest versions of Flask and Python installed.
+
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+
```

Comment:
0. Create a new Miro App and make sure to check the box for `expire authorization token` as shown in the screenshot below.

<img width="434" alt="expire-auth-token" src="https://github.com/miroapp/app-examples/assets/10428517/d479bf68-a142-4d02-8c4e-fc8cfa816b95">